scripts/coverage.sh

```diff
@@ -6,7 +6,7 @@ log() {
   echo "$*" >&2
 }
 
-SOLIDITY_COVERAGE=true scripts/test.sh || log "Test run failed"
+npx truffle run coverage --network development
```

Comment:
Is this the syntax for running a truffle plugin?

Reply:
Yes. There's more at the [truffle docs for plugins](https://www.trufflesuite.com/docs/truffle/getting-started/writing-external-scripts#third-party-plugin-commands).